src/css/custom.css

```diff
@@ -0,0 +1,81 @@
+body{
+  background-color: ghostwhite;
+}
+
+.title {
+  margin: 35px 55px;
+}
+
+.new-book {
+  margin: 35px 55px;
+}
+
+.form fieldset{
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.title-books ul{
+  list-style: none;
+  display: flex;
+  margin: 20px 20px;
+}
+
+.title-books li{
+  margin: 15px 60px;
+}
+
+.books ul{
+  list-style: none;
+  display: flex;
+  margin: 20px 20px;
+}
+
+.books li{
+  margin: 15px 60px;
+}
+
+.size1-column {
+  width: 5%;
+  /*text-align: center;*/
```

Comment:
Please remove commented out code!
As can be seen from [this screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
It is better to delete commented code. If you need to review a previous version of your code, use git history. 🙂